pkg/kubelet/container/testing/mockfileinfo.go

```diff
@@ -27,83 +27,97 @@ import (
 	gomock "github.com/golang/mock/gomock"
 )
 
-// Mock of FileInfo interface
+// MockFileInfo is a Mock of the FileInfo interface
 type MockFileInfo struct {
 	ctrl     *gomock.Controller
-	recorder *_MockFileInfoRecorder
+	recorder *MockFileInfoRecorder
 }
 
-// Recorder for MockFileInfo (not exported)
-type _MockFileInfoRecorder struct {
+// MockFileInfoRecorder is a Recorder for MockFileInfo
+type MockFileInfoRecorder struct {
```

Comment:
should this be lowercase so that it's not exported?

Reply:
Absolutely agree! Fixed it.